pilot/proxy/envoy/testdata/lds-v0-none-auth-migration.json.golden

```diff
@@ -0,0 +1,1002 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15001",
+    "name": "virtual",
+    "filters": [],
+    "bind_to_port": true,
+    "use_original_dst": true
+   },
+   {
+    "address": "tcp://0.0.0.0:443",
+    "name": "http_0.0.0.0_443",
```

Comment:
I don't think this is right. We don't have any 443 listeners anymore. @ayj correct?